src/components/NestedScrollView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ScrollView, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Any children to display */
+    children: PropTypes.node,
```

Comment:
Does this pass for all types of children?

Reply:
You could have the FlatList at any n-the level child and hence I've added a generic node.

Reply:
Is Node a generic type? I will check it.